src/modules/postCreation/store/postCreationSlice.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { PostTypeEnum } from '../../../lib/types';
+import { ICheckboxes } from '../PostTopicSelector';
+
+interface INewPostDraft {
+  topics: ICheckboxes;
+  title?: string;
+  htmlContent: string;
+  preview: string;
+}
+
+export interface IPostCreationState {
+  [PostTypeEnum.ARTICLE]: INewPostDraft;
+  [PostTypeEnum.DOPYS]: INewPostDraft;
+}
+
+const initialState: IPostCreationState = {
+  [PostTypeEnum.ARTICLE]: {
+    topics: {},
+    title: '',
+    htmlContent: '',
+    preview: '',
+  },
+  [PostTypeEnum.DOPYS]: {
+    topics: {},
+    htmlContent: '',
+    preview: '',
+  },
+};
+
+export const postCreationSlice = createSlice({
+  name: 'postCreation',
+  initialState,
+  reducers: {
+    // saveNewPostDraft: (state, action: PayloadAction<IPostCreationState>) => {
+    //   state = action.payload;
+    // },
```

Comment:
remove if not needed

Reply:
left this for comparison purpose, i'll remove commented sections before merge